apps/cwl_to_slack/main.tf

```diff
@@ -71,21 +71,15 @@ resource "aws_iam_role_policy" "slack_notifier_logs" {
 EOF
 }
 
-data "archive_file" "lambda_zip" {
-  type = "zip"
-  source_dir = "./target"
-  output_path = "./lambda.zip"
-}
-
 resource "aws_lambda_function" "slack_notifier" {
-  function_name = "${var.app_name}"
-  filename = "${data.archive_file.lambda_zip.output_path}"
+  function_name = "cloudwatch-slack-notifier"
   description = "An Amazon SNS trigger that sends CloudWatch alarm notifications to Slack."
-  runtime = "nodejs6.10"
-  handler = "index.handler"
-  memory_size = 128
+  s3_bucket = "org-hca-logs-lambda-deployment"
+  s3_key = "cwl_to_slack/cloudwatch-slack-notifications_App.zip"
```

Comment:
Here too, I would parameterize the s3 info so there is one source of truth

Reply:
👍